File: 백준/Bronze/11720. 숫자의 합/숫자의 합.c
#include <stdio.h>
int main(void) {
  int a, b; int sum=0;
  scanf("%d", &a);
  for(int i=0;i<a;i++){
    scanf("%1d", &b);
    sum += b;
  }
    printf("%d\n", sum);
  
  return 0;
}

File: 백준/Bronze/11720. 숫자의 합/숫자의 합.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException{

    	BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    	int a = Integer.parseInt(br.readLine());
		String b = br.readLine();
		br.close();
		
		int sum = 0;
        
		for(int i = 0; i < a; i++) {
			sum += b.charAt(i)-'0';
		}
		System.out.print(sum);
    }
}